posthog/api/sharing.py

```diff
@@ -223,6 +237,52 @@ def retrieve(self, request: Request, *args: Any, **kwargs: Any) -> Any:
         return render_template(
             "exporter.html",
             request=request,
-            context={"exported_data": json.dumps(exported_data, cls=DjangoJSONEncoder)},
+            context={
+                "exported_data": json.dumps(exported_data, cls=DjangoJSONEncoder),
+                "asset_title": asset_title,
+                "add_og_tags": add_og_tags,
+            },
             team_for_public_context=resource.team,
         )
+
+    def _exported_asset_is_stale(self, exported_asset: ExportedAsset | None) -> bool:
+        if not exported_asset:
+            return True
+
+        export_age = now() - cast(datetime, exported_asset.created_at)
+        return export_age.total_seconds() > 3600 * 3
```

Comment:
Could possibly use `export_age > timedelta(hours=1) ` to make this clearer

Reply:
👍 I'll cover the point on using `first` and this in https://github.com/PostHog/posthog/pull/16324 later today